index.html

```diff
@@ -16,13 +16,13 @@
         <link rel="stylesheet" href="/grid.css" />
         <title>Craigslist Home Page</title>
     </head>
-
+    <!--Try this assignment again!!-->
     <body>
         <header>
             <img src="/assets/html_css_js.png" />
             <h1>Craigslist Copy</h1>
         </header>
-
+        <nav>craigslist</nav>
```

Comment:
You want to avoid using ```<br>``` tags — they're unnecessary these days with layout solutions like grid and flex